schema/m3-region.schema.json

```diff
@@ -1359,6 +1359,45 @@
                 }
               }
             }
+          },
+          "reusableRoomwideNotable": {
+            "$id": "#/properties/rooms/items/properties/reusableRoomwideNotable",
+            "type": "array",
+            "title": "Very similar notable strats that exist within multiple links within a single room.",
+            "items": {
+              "$id": "#/properties/rooms/items/properties/reusableRoomwideNotable/items",
+              "type": "reusableRoomwideNotable",
+              "title": "Reusable Roomwide Notable Strat",
+              "required": [
+                "name",
+                "note"
+              ],
+              "additionalProperties": false,
+              "properties": {
+                "name": {
+                  "$id": "#/properties/rooms/items/properties/reusableRoomwideNotable/items/properties/name",
+                  "type": "string",
+                  "title": "Shared Reusable Roomwide Strat Name",
+                  "default": "",
+                  "examples": [
+                    "Bubble Mountain Damage Boost"
+                  ],
+                  "pattern": "^(.*)$"
+                },
+                "note": {
+                  "$id": "#/properties/rooms/items/properties/reusableRoomwideNotable/items/properties/note",
+                  "type": "string",
```

Comment:
On the one hand `note` will be used just like all other notes so this could just be a `$ref`.
On the other, this has a clearer purpose in describing the connected strat.